ux.symfony.com/src/Service/RecipeParser.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Service;
+
+class RecipeParser
+{
+    public function getTitle(string $content): string
```

Comment:
thought: Since $content is the same argument (name and value), maybe we could have it as a constructor and class attribute.